### DISCUSSION TITLE
Possible to have a Map with custom georeferenced image for background?

### DISCUSSION THREAD

**Discussion by jameslavery** [dotnet/maui#17455] [Q&A]
I'm investigating a navigation app which will need to display nautical charts with the current location on them. The charts are (I assume) georeferenced images.

Is it possible to use a georeferenced image in a layer on a map?

**Comment by madmapper** [comment]
The ESRI Maps SDK for .net supports MAUI and would allow you to accomplish what you want to do at the Lite license level depending on the data formats you want to use.
https://developers.arcgis.com/net/

**Reply by jameslavery** [reply]
@madmapper - Thanks. I think there's also an option on Android, providing my own tiles with a TileOverlay. Of course I'd have to do the equivalent on iOS, too. So a compromise between out of the box for $$$ or rolling my own for "free"
